src/libstd/fs.rs

```diff
@@ -81,9 +81,18 @@ use time::SystemTime;
 /// # }
 /// ```
 ///
+/// Note that, although read and write methods require a `&mut File`, because
+/// of the interfaces for [`Read`] and [`Write`], it is still possible to
+/// modify a file through a `&File`, either through methods that take `&File`
+/// or by retrieving a raw OS filehandle and modifying the file that way.
+/// Additionally, many operating systems allow concurrent modification of files
+/// by different processes. Care should be taken not to assume that holding a
+/// `&File` means that the file will not change.
```

Comment:
This is a bit of a nitpick, but could you please phrase this in active voice?

"..., the holder of a `&File` can still modify that file, either through methods ..."

"... Avoid assuming that holding a `&File` means ..."